_data/task.yml

```diff
@@ -9,8 +9,10 @@ websites:
     img: airtable.png
     tfa:
       - sms
+      - phone
```

Comment:
The [documentation](https://support.airtable.com/hc/en-us/articles/219409917) says call my phone is a 2FA option, but it isn't for me. 
![airtable com](https://user-images.githubusercontent.com/10876578/73505803-7b0b9480-43cb-11ea-9c25-c7d48e994fbf.png)

Reply:
That’s odd, not sure why that is.